build.sbt

```diff
@@ -58,12 +58,13 @@ lazy val higherKindCore = projectMatrix
     defaultSettings,
     scala3MigratedModuleOptions,
     name := "tofu-core-higher-kind",
+    libraryDependencies ++= Seq(catsCore, catsFree, catsTaglessCore),
     libraryDependencies ++= {
       CrossVersion.partialVersion(scalaVersion.value) match {
         case Some((2, n)) =>
-          Seq(catsCore, catsFree, catsTaglessMacros)
+          Seq(catsTaglessMacros)
         case _            =>
-          Seq(catsCore, catsFree, catsTaglessCore)
```

Comment:
cats-tagless https://github.com/typelevel/cats-tagless/releases/tag/v0.16.0 btw

Reply:
seems like module cats-tagless-macros remaines only for Scala 2. Macros for Scala 3 have become available in the cats-tagless-core module since version 0.16.
Sounds like it is time to start working on HigherKindedMacros 😅🔫

Reply:
That's right! We need to start working on HKT Macro.

All works except type members in the interfaces 🙁